Spanner/src/Batch/BatchClient.php

```diff
@@ -119,6 +119,11 @@ class BatchClient
      */
     private $databaseName;
 
+    /**
+     * @var string
```

Comment:
Could we please document that this could also accept null?

Reply:
Sure, will add in a new PR